test/shell/exec-usergroup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -x
+
+if ! timeout 1s sudo -A true; then
+	echo "sudo disabled: not checking exec user and group"
+	exit
+fi
+
+function setup {
+	mkdir -p /tmp/mgmt/test-exec-usergroup/
+	sudo -A chown nobody:$group /tmp/mgmt/test-exec-usergroup/
+	sudo -A chmod 775 /tmp/mgmt/test-exec-usergroup/
```

Comment:
please use the symbolic format, eg: `chmod ug=rwx,o=rx /tmp/mgmt/test-exec-usergroup/`